python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -606,40 +624,55 @@ def helptext(f, s):
 
     def _init(self, *args, **kwds):
 
-        # The first thing we do is, if we were called with GPIProxyClassFactory(_existing=some_ganga_object)
-        # to simply create a link between them and us and nothing else.
-        if '_existing' in kwds:
-            # Link the two together in the proxy layer
-            setattr(self, implRef, kwds['_existing'])
-            setattr(kwds['_existing'], proxyObject, self)
-            return
+        ## Zero-th fully initialize self before moving on
+        super(type(self), self).__init__()
```

Comment:
This is potentially dangerous as any subclasses of a proxy class would then enter an infinite loop. Admittedly we don't ever subclass a proxy class but in fact this would prevent it.